apps/windows/metro.config.js

```diff
@@ -11,9 +11,6 @@ const {getWatchFolders} = require('@uifabricshared/build-native');
 module.exports = {
   watchFolders: getWatchFolders(),
   resolver: {
-    resolveRequest: require('react-native-windows/metro-react-native-platform').reactNativePlatformResolver(
-      {windows: 'react-native-windows'},
-    ),
```

Comment:
For my own understanding, is there no more custom code needed to support out of tree platforms 0.62 onwards? Asking for when we move the macOS branch to 0.62 and I need to make similar changes.

Reply:
0.61 removed the haste map which greatly simplified things, then fairly recently Coates made a change to how the base metro configs were set up that made out of tree platforms far easier to author. Mac should be able to take advantage of that once it moves.

Reply:
For more clarity here, I think that as long as you are referencing the latest version of the @react-native-community/cli project and possibly metro you should get the appropriate changes to make this easy.